frontend/src/components/search/components/filter-bar/filter-bar.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { FC } from 'common/types/types';
+import { IconName, FilterDate, FilterType } from 'common/enums/enums';
+import { Icon } from 'components/common/common';
+
+import styles from './styles.module.scss';
+
+type Props = {
+  filterType: string;
+  onChangeFilterType: (value: string) => void;
+  filterDate: string;
+  onChangeFilterDate: (value: string) => void;
+};
+
+const FilterBar: FC<Props> = ({ filterType: fType, onChangeFilterType, filterDate: fDate, onChangeFilterDate }) => {
+  const onHandleChangeFilterType = (e: React.FormEvent<HTMLSelectElement>): void =>
+    onChangeFilterType(e.currentTarget.value);
+  const onHandleChangeFilterDate = (e: React.FormEvent<HTMLSelectElement>): void =>
+    onChangeFilterDate(e.currentTarget.value);
+
+  return (
+    <>
+      <div className={styles['filter-bar']}>
+        <div className={styles['filter-bar-type']}>
+          <label htmlFor="filterBarType" className={styles['filter-bar-select']}>
+            <select id="filterBarType" onChange={onHandleChangeFilterType} value={fType}>
+              <option value="">all</option>
```

Comment:
Empty string is not enough understandable that it means "all"

Better to use enums in component wrapper.

In general I am not sure we need this option. Maybe better to add X icon to remove selection to leave select empty

Reply:
@kyrylo-burmelov This component only for mobile, reference youtube:
![image](https://user-images.githubusercontent.com/15041213/184387173-bdc866d4-47ec-48ff-945f-6573d93047da.png)

Reply:
@kyrylo-burmelov If there will be 'X', users don't understand, what type filter is.
Also on desktop items 'all' and 'anytime' should not be present, so I use another component.

Reply:
Agreed. Just create a specific value for this. Empty string as filter id is not good